Tools/WebKitTestRunner/InjectedBundle/TestRunner.cpp

```diff
@@ -888,6 +888,7 @@ void TestRunner::denyWebNotificationPermissionOnPrompt(JSStringRef origin)
 void TestRunner::removeAllWebNotificationPermissions()
 {
     WKBundleRemoveAllWebNotificationPermissions(InjectedBundle::singleton().bundle(), page());
+    postSynchronousPageMessage("ResetNotificationPermissions");
```

Comment:
Can we use `RemoveAllWebNotificationPermissions` to match the name of the function? I believe this is the pattern in our existing code and it makes it clearer what caused this IPC.

Reply:
Fixed!